### PR TITLE
Metrics-webapp

### PR DESCRIPTION
**Pull request by ONJoseph**
Day 1
- [ ] Create react app.
- [ ] Select corona.lmao.ninja API.
- [ ] Build files structure for your React app.
- [ ] Prepare routes and navigation in your app.
Day 2
- [ ] Add functionality for a user to display a list of items and filter them and connect data from the corona.lmao.ninja API.
- [ ] Create the tests for the application.
Day 3
- [ ] Style components to match the design provided.
- [ ] Deploy the project and test for final details.
- [ ] Record a video for the project.
- [ ] Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,128 @@
...
+
+### Usage
+
+To run the project, execute the following command:
+
+- Clone the repository using `https://github.com/ONJoseph/metrics-webapp.git`
+- Change directory into the project folder: `cd metrics-webapp`
+- Run `npm install`
+- Run `npm start`
+- A new browser will open automatically with application loaded
```
Comment (Nemwel-Boniface) [inline README.md]: - [x] I have double checked in both your README and the pull request summary and I have not seen the link to the project video demo. Kindly take it and include it in your project.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @ONJoseph       :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have an amazing project here! I am actually amazed and I enjoyed doing this review :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ONJoseph) [reply README.md]: Hi @Nemwel-Boniface , Thanks a lot for the review! I have added the link to the project documentation video. Best regards!

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ONJoseph , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
> 

Hi @Nemwel-Boniface, Many Thanks. Cheers!

**Pull request merged**